packages/site/src/components/PropsList.tsx

```diff
@@ -107,7 +111,7 @@ export const PropsList = ({
                         {item.required && <InlineLabel>Required</InlineLabel>}
                       </div>
                     </Grid.Cell>
-                    <Grid.Cell size={{ xs: 6 }}>{item.component}</Grid.Cell>
+                    <Grid.Cell size={{ xs: 12 }}>{item.component}</Grid.Cell>
```

Comment:
As a "todo" I would like to render this as perhaps a small/subdued Text element, but DataList is really finicky to pass in custom components in my experience.